index.html

```diff
@@ -270,7 +269,7 @@ <h1>Attempts</h1>
             <input type="text" id="player2_input" onfocus="this.value=''" value="Player 1">
         </div>
 
-        <button id="playButton" onclick="">Play Now</button>
+        <button id="playButton" type="button">Play Now</button>
```

Comment:
Does this button work when clicked? I am testing it in JS bin and it's not functioning